### PR TITLE
add info about s3 default

### PR DESCRIPTION
**Pull request by AlexicaWright**

### CODE REVIEW

File: modules/ROOT/pages/clustering/databases.adoc [hunk 1]
```diff
@@ -156,6 +156,8 @@ The sources of seeds are called _seed providers_.
 
 The mechanism is pluggable, allowing new sources of seeds to be supported (see link:https://www.neo4j.com/docs/java-reference/current/extending-neo4j/project-setup/#extending-neo4j-plugin-seed-provider[Java Reference -> Implement custom seed providers] for more information).
 The product has built-in support for seed from a mounted file system (file), FTP server, HTTP/HTTPS server and Amazon S3.
+The S3 is supported by default, but the other providers require configuration of xref:reference/configuration-settings.adoc#config_dbms.databases.seed_from_uri_providers[`dbms.databases.seed_from_uri_providers`].
```
Comment (davidoliverSP2) [inline modules/ROOT/pages/clustering/databases.adoc:159]:
```suggestion
S3 is supported by default, but the other providers require configuration of xref:reference/configuration-settings.adoc#config_dbms.databases.seed_from_uri_providers[`dbms.databases.seed_from_uri_providers`].
```

**Review by davidoliverSP2 — APPROVED** [review]
Thanks @AlexicaWright 

This seems like an important clarification/addition. Did you consider adding this in a _Note_ or _Important_ admonition?

File: modules/ROOT/pages/clustering/databases.adoc (hunk 1 above)
Comment (AlexicaWright) [reply modules/ROOT/pages/clustering/databases.adoc:159]: Thanks David, omitted _provider_ here. Better to omit _The_ as well.

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-646.surge.sh

**Pull request merged**